tests/cpp/test_sdpa_node.cpp

```diff
@@ -9,25 +9,36 @@
 #include <gtest/gtest.h>
 
 #include <fusion.h>
-#include <id_model/id_model.h>
-#include <mma_type.h>
 #include <ops/all_ops.h>
+#include <ops/utils.h>
 #include <preseg_passes/allocation_order_inference.h>
+#include <preseg_passes/move_split_cat.h>
 #include <preseg_passes/optimization_pass.h>
-#include <scheduler/all_schedulers.h>
-#include <scheduler/mma_utils.h>
 #include <tests/cpp/utils.h>
 #include <tests/cpp/validator.h>
 
 namespace nvfuser {
 
-using SDPATest = NVFuserTest;
+class SDPATest : public NVFuserTest {
+ protected:
+  SDPATest() : optimization_guard_(false), allocation_order_guard_(false) {}
+
+ private:
+  // Note: `MoveSpliCat` and `AllocationDomain` preseg passes use ID model.
+  // `SdpaOp` currently does not work with ID model since it requires all
```

Comment:
```suggestion
  // Note: `MoveSplitCat` and `AllocationDomain` preseg passes use ID model.
  // `SdpaFwdOp` currently does not work with ID model since it requires all
```